src/nom.rs

```diff
@@ -602,9 +602,10 @@ pub fn float(input: &[u8]) -> IResult<&[u8],f32> {
     recognize!(
       tuple!(
         opt!(alt!(tag!("+") | tag!("-"))),
-        alt!(
-          delimited!(digit, tag!("."), opt!(digit))
-          | delimited!(opt!(digit), tag!("."), digit)
+        alt_complete!(
```

Comment:
Are there tests for streaming?  Something like a test_parser (parser, "some data") that tests the parser with a single buffer with the whole string, and with a streaming wrapper that feeds it 1 byte at a time?